script.js

```diff
@@ -1,12 +1,116 @@
-//You can edit ALL of the code here
+//TV Show Project
+
+const allEpisodes = getAllEpisodes();
+const episodeContainer = document.getElementById("cardContainer");
+const rootElem = document.getElementById("root");
+const searchInput = document.getElementById("searchEpisode");
+const episodesCount = document.getElementById("searchResult");
+const viewAllEpisodesButton = document.querySelector("#viewAllEpisodes");
+
 function setup() {
-  const allEpisodes = getAllEpisodes();
   makePageForEpisodes(allEpisodes);
 }
 
+// display all episodes //
+
 function makePageForEpisodes(episodeList) {
-  const rootElem = document.getElementById("root");
-  rootElem.textContent = `Got ${episodeList.length} episode(s)`;
+  rootElem.innerHTML = "";
+  episodeList.forEach((episode) => createCard(episode));
+  episodesCount.innerText = `Displaying ${episodeList.length}/${allEpisodes.length}`;
+}
+
+// create card //
+
+function createCard(episode) {
+  let cardDiv = document.createElement("div");
+  cardDiv.classList.add("cardDiv");
+  let cardHeader = document.createElement("div");
+  cardHeader.classList.add("cardHeader");
+  let episodeTitle = document.createElement("h2");
```

Comment:
I know we haven't touched upon in it classes, but this is known as 'header stepping'. In a document. we have the headers running from 1-6. It's best practice to put them in order and style as you need to. For example, your card would be better off having a h2 title and either another h2 for the season or a h3 for the season. Doing this makes pages and sites more accessible for assistive technology~